hokusai/cli/review_app.py

```diff
@@ -0,0 +1,53 @@
+import click
+
+import hokusai
+
+from hokusai.cli.base import base
+from hokusai.cli.staging import KUBE_CONTEXT
+from hokusai.lib.common import set_verbosity, CONTEXT_SETTINGS, clean_string
+from hokusai.lib.config import config
+from hokusai.lib.namespace import create_new_app_yaml
+
+@base.group()
+def review_app(context_settings=CONTEXT_SETTINGS):
+  """Manage/Create review apps"""
+  pass
+
+@review_app.command(context_settings=CONTEXT_SETTINGS)
+@click.argument('app_name', type=click.STRING)
+@click.option('-v', '--verbose', type=click.BOOL, is_flag=True, help='Verbose output')
+@click.option('-sf', '--source-file', type=click.STRING, default='hokusai/staging.yml', help='Source deployment file')
+@click.option('-dn', '--destination-namespace', type=click.STRING, default=None, help='Target Namespace')
+def setup(app_name, verbose, source_file, destination_namespace):
```

Comment:
Minor but the click library will [automatically parse](http://click.pocoo.org/5/documentation/) any `@click.command`-wrapped function docstrings and display them as `--help` :)